src/ServicePulse.Host/vue/src/components/TimeSince.vue

```diff
@@ -1,21 +1,18 @@
-﻿<script setup>
+﻿<script setup lang="ts">
 import { onMounted, onUnmounted, ref } from "vue";
 import moment from "moment";
 
-const props = defineProps({
-  dateUtc: String,
-  default: function () {
-    return "0001-01-01T00:00:00";
-  },
-});
+const emptyDate = "0001-01-01T00:00:00";
+
+const props = withDefaults(defineProps<{ dateUtc: string }>(), { dateUtc: emptyDate });
 
-let interval = null;
+let interval: number | undefined = undefined;
```

Comment:
I left this one as `undefined` @PhilBastian, mostly because the `window.clearInterval` takes a number or undefined